indico/modules/designer/pdf.py

```diff
@@ -127,10 +127,11 @@ def _draw_item(self, canvas, item, tpl_data, content, margin_x, margin_y):
         item_y = float(item['y']) / PIXELS_CM * cm
         item_width = item['width'] / PIXELS_CM * cm
         item_height = (item['height'] / PIXELS_CM * cm) if item.get('height') is not None else None
+        item_preserve_aspect_ratio = item.get('preserve_aspect_ratio', True)
```

Comment:
Shouldn't this also be included in the `TEMPLATE_DATA_JSON_SCHEMA` JSON schema so that key won't get rejected when sent by the designer JS code (even if our own code never sends it, it's still valid)?

Reply:
We're also injecting the `preserve_aspect_ratio` through another signal in the code, so we don't really need to validate it in the schema

Reply:
Added